ANTsRegistration/ANTsRegistration.py

```diff
@@ -1310,6 +1642,71 @@ def buildTemplate(
                 background=outputTemplate, fit=True, rotateToVolumePlane=True
             )
 
+    def installANTsPyX(self):
+
+        try:
+            import ants
+        except:
+            slicer.util.pip_install('antspyx')
+
+
+    def generateJacobian(self, pathList, templateNode, templateMaskNode, covariatesFilePath, rformula, outputImageNode):
```

Comment:
@muratmaga This code is pretty much verbatim from the example you gave me, but hangs on the call to `ants.ilr`

Reply:
Does it work as pure python? I.e is crash slicer specific or reproduces?

Reply:
Here's a test script using the mouse ct data: https://gist.github.com/sjh26/d3c0e7a6be1d9d8ffe2e5ea9919ba556.  It just hangs on my system.